plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/fakes/TestSpanAddingParameters.kt

```diff
@@ -42,29 +28,10 @@ class TestSpanAddingParameters(
             .convention(setOf(InstrumentationFeature.FILE_IO, InstrumentationFeature.DATABASE))
 
     override val sdkStateFile: RegularFileProperty
-        get() = filePropertyFactory.newFileProperty().value(filePropertyFactory.file(_sdkStateFile))
+        get() = TODO()
```

Comment:
What's the relation of this change with this PR related to gradle-build caching, and Why is it a TODO now?

Reply:
True, the PR could be named better. I initially opened it to make the CI green, and this was one of the reasons it was failing (we were using some internal Gradle APIs that were trying to create a temp folder). After checking I realized that this property is not used in tests at all, so I just dropped a TODO to make it compile. It could also be a `mock()` if you prefer.

Reply:
Gotcha, a `mock` would make more sense since we know there's nothing to be done, TODO gives us the impression we've to do something about it, all good then, it was just to understand the change, happy to keep as it is as well, your call.

Reply:
will change later when I touch this again 👍